eva_submission/eload_preparation.py

```diff
@@ -154,14 +159,25 @@ def detect_metadata_attributes(self):
     def find_genome(self):
         scientific_name = self.eload_cfg.query('submission', 'scientific_name')
         analyses = self.eload_cfg.query('submission', 'analyses')
+
+        contig_alias_url, contig_alias_user, contig_alias_pass = get_contig_alias_db_creds_for_profile(
+            cfg['maven']['environment'], cfg['maven']['settings_file'])
+
+        payload = []
```

Comment:
I'd name this something like `contig_alias_payload` just to make it clear, since this method is doing a few different things...

Reply:
This is implemented with the new changes